super_editor/lib/src/default_editor/document_gestures_mouse.dart

```diff
@@ -504,6 +504,12 @@ Updating drag selection:
     }
   }
 
+  bool _getIsDragginBack(Offset baseOffset, Offset extentOffset) {
```

Comment:
We should be using node indexes instead of coordinates so it has a chance of doing the right thing in RTL text. The decision tree is just

- extent node index < base node index? dragging upstream
- extent node index > base node index? dragging downstream
- extent position offset < base position offset? dragging upstream
- otherwise, dragging downstream

You can get the indices of the nodes from `widget.document`.